.github/workflows/namespace-notifier.yml

```
@@ -0,0 +1,18 @@
+name: Buildpack Registry Namespace Notifier
+on: 
+  push:
+    branches:
+      - slack-test
```

```suggestion
      - main
```

Let's go ahead and set this to main when we merge it. The action will fail at first, but that's fine.